OpenRA.Game/Graphics/SequenceProvider.cs

```diff
@@ -22,16 +22,12 @@ namespace OpenRA.Graphics
 	public interface ISpriteSequence
 	{
 		string Name { get; }
-		int Start { get; }
 		int Length { get; }
-		int Stride { get; }
 		int Facings { get; }
 		int InterpolatedFacings { get; }
 		int Tick { get; }
 		int ZOffset { get; }
-		int ShadowStart { get; }
 		int ShadowZOffset { get; }
-		int[] Frames { get; }
```

Comment:
What is the criteria for what gets removed and what stays here?

Reply:
Internal implementation details about how sprites are laid out within their source files are removed, things that are relevant to their usage by the game stay.